pkg/buildpack/multi_architecture_helper.go

```diff
@@ -73,10 +73,13 @@ func CopyConfigFile(baseDir string, target dist.Target, distroVersion string) (s
 //   - Given a target linux/amd64 the platform root folder will be <root>/linux/amd64 if the folder exists
 //   - Given a target windows/amd64:windows@10.0.20348.1970 the platform root folder will be <root>/windows/amd64/windows@10.0.20348.1970 if the folder exists
 //   - When no target folder exists, the root folder will be equal to <root> folder
-func PlatformRootFolder(root string, target dist.Target, version string) (bool, string) {
-	targets := target.ValuesAsSlice(version)
-	pRootFolder := root
+func PlatformRootFolder(bpPathURI string, target dist.Target) (bool, string) {
```

Comment:
I assume the given target has only 1 distribution, maybe I need to put that in the comments or add some validation.

Reply:
Do we allow multiple distributions per target? Is distribution always required?

Reply:
- Yes, we allow multiple distributions per target
- No, it is not always required. For example: `linux/amd64` will not have distributions, but `linux/amd64/v2/ubuntu@18.01` will.

That's the reason I added the `dist.ExpandTargetsDistributions()` method, and we can safely call `PlatformRootFolder`, that's the actual comment I am referring to

Reply:
Gotcha, okay that makes sense. I think a comment would be helpful

Reply:
Added